### PR TITLE
VP-4128: Create  workflow

### PR DESCRIPTION
**Pull request by krankenbro**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -23,270 +23,82 @@ on:
...
+      run: vc-build Compile
 
-    - name: Install VirtoCommerce.GlobalTool
-      run: dotnet tool install --global VirtoCommerce.GlobalTool
+    - name: Unit Tests
+      run: vc-build Test -skip
 
-    - name: Download a Build Artifact
-      uses: actions/download-artifact@v2
+    - name: SonarCloud End
```
Comment (yecli) [inline .github/workflows/main.yml:65]: Is it ok that it is the same as 
name: SonarCloud Begin?

Comment (krankenbro) [reply .github/workflows/main.yml:65]: Thanks! it is typo

**Review by yecli — APPROVED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -23,270 +23,82 @@ on:
...
+      uses: VirtoCommerce/vc-github-actions/sonar-scanner-end@dev
+
+    - name: Packaging
+      run: vc-build Compress -skip Clean+Restore+Compile+Test
+
+    #- name: Swagger Schema Validation
```
Comment (yecli) [inline .github/workflows/main.yml:71]: Is it OK that Swagger Schema Validation is commented out?

Comment (krankenbro) [reply .github/workflows/main.yml:71]: It is temporary  because of https://virtocommerce.atlassian.net/browse/VP-4255

**Pull request merged**